public/angular/templates/reports/index.html

```diff
@@ -25,7 +25,7 @@
               <div class="form-group col mb-0">
                 <div class="input-group input-group-sm">
                   <input type="text" ng-model="searchParams.tags" aria-label="tagSearch" class="form-control filters__input" placeholder="{{'Enter Tags' | translate}}">
-                  <div class="input-group-append" ng-click="clearTags()">
+                  <div class="input-group-append">
```

Comment:
Is this the actual change that fixed the Go button? Simple!

Reply:
Yea. Loll